managed/services/telemetry/config.default.yml

```diff
@@ -720,3 +720,86 @@ telemetry:
         label: "job"
       - metric_name: "6"
         value: 1
+
+  # Backup Management information
+  - id: BackupManagementStorageTypes
+    source: PMMDB_SELECT
+    query: type AS storage_type, count(*) AS count FROM backup_locations GROUP BY type
+    transform:
+      type: JSON
+      metric: backup_management_storage_types
+    summary: "How many storages configured by types"
+    data:
+      - metric_name: "1"
+        column: "storage_type"
+      - metric_name: "2"
+        column: "count"
+
+  - id: BackupManagementScheduledTasks
+    source: PMMDB_SELECT
+    query: type AS task_type, count(*) AS count FROM scheduled_tasks GROUP BY task_type
+    transform:
+      type: JSON
+      metric: backup_management_scheduled_tasks
+    summary: "How many backup tasks scheduled by types"
+    data:
+      - metric_name: "1"
+        column: "task_type"
+      - metric_name: "2"
+        column: "count"
+
+  - id: BackupManagementJobsPerformed
+    source: PMMDB_SELECT
+    query: >
+      j.type bm_job_type,
+      	coalesce(j.mod, 'N/A') bm_backup_mode,
+      	coalesce(j.dm, 'N/A') bm_data_model,
+      	coalesce(a_backup.type, 'N/A') bm_job_run_type,
+      	j.job_status bm_job_status,
+      	coalesce(bl_backup.type, bl_restore.type, 'N/A') bm_location_type,
+      	count(*) count
+      from (select type, 
+      			"data"->"type"->>'mode' mod, 
+      			"data"->"type"->>'data_model' dm, 
+      			"data"->"type"->>'artifact_id' artifact_id,
+      			"data"->"type"->>'restore_id' restore_id,
+      			case error when '' then 'success' else 'failed' end job_status
+      		from jobs 
+      		where created_at BETWEEN NOW() - INTERVAL '24 HOURS' AND NOW() 
+      			group by type, mod, dm, artifact_id, restore_id, job_status) j
+      left join artifacts a_backup
+      	on a_backup.id = j.artifact_id
+      left join backup_locations bl_backup
+      	on bl_backup.id = a_backup.location_id 
+      left join restore_history rh
+      	on rh.id = j.restore_id
+      left join artifacts a_restore
+      	on a_restore.id = rh.artifact_id
+      left join backup_locations bl_restore
+      	on bl_restore.id = a_restore.location_id
+      group by 
+      	bm_job_type, 
+      	bm_backup_mode, 
+      	bm_data_model, 
+      	bm_job_run_type, 
+      	bm_job_status, 
+      	bm_location_type
+    transform:
+      type: JSON
+      metric: backup_management_jobs_performed
+    summary: "Backup/restore jobs sliced by different parameters"
+    data:
+      - metric_name: "1"
+        column: "bm_job_type"
+      - metric_name: "2"
+        column: "bm_backup_mode"
+      - metric_name: "3"
+        column: "bm_data_model"
+      - metric_name: "4"
+        column: "bm_job_run_type"
+      - metric_name: "5"
+        column: "bm_job_status"
+      - metric_name: "6"
+        column: "bm_location_type"
+      - metric_name: "7"
+        column: "count"
```

Comment:
Some metrics in this file has sting name, some numeric. I don't remember what was the reason to use numeric names?

Reply:
As I understood, numbers used when we choose json format.

Reply:
it has to be numeric because JSON will be sorted by keys (it is default behavior with serialization), plus you will be to access in materialized view by number, you will need to declare ([example](https://github.com/percona-platform/telemetryd/blob/main/migrations/011_api_usage_view.up.sql#L31)), like this one `Array(Tuple(String,String,String,String,String,String,String))` and you will refer to to them by number.
Another concern is storage size.
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/91831719/217799859-f2b6862b-96f5-47a4-b56d-f3491ed49ca7.png">